frontend/src/User/UserProfile/UserProfile.tsx

```diff
@@ -112,7 +112,7 @@ const UserProfile: React.FC = () => {
   };
 
   let name;
-  if (role === 'admin' || role === 'lead') {
+  if (role === 'lead') {
     name = (
       <Form.Group widths="equal">
         <Form.Input
```

Comment:
Technically this is a behavior change. But it doesn't really better, since if you really have admin permission, you can edit it in the user list any ways.